.github/copilot-instructions.md

```diff
@@ -50,6 +50,10 @@ SvelteKit is a web application framework for building modern web applications wi
 4. `cd packages/kit && pnpm test:unit` - Run unit tests
 5. **For changes to @sveltejs/kit**: `cd packages/kit && pnpm prepublishOnly` - Generate types
 
+#### Add changeset once you're done
+
+Run `npx changeset` then follow the prompts. Use this after having finished the task. Most of the time this is a patch release for `@sveltejs/kit`. Use a short and descriptive message, e.g. `fix: run load functions in order`
```

Comment:
```suggestion
Run `npx changeset` then follow the prompts. Use this after having finished the task. Most of the time this is a patch release for `@sveltejs/kit`. Use a short and descriptive message, e.g. `fix: run load functions in order`. Always prefix the message with either `fix`, `feat`, `breaking`, or `chore`.
```
Wonder if it's better to give it a list of options (and maybe explain when to use which) but I think we'll probably only use this for fixes.